org/urls.py

```diff
@@ -4,15 +4,13 @@
 
 urlpatterns = [
     path('', OrgView.as_view()),
-    path('<int:org_id>/volunteer/',AddVolunteer,name='add_volunteer'),
-    path('<int:org_id>/group/', GroupView.as_view()),
-    path('<int:org_id>/', EditOrg, name='edit_org'),
+    path('<int:org_id>/', EditOrgView.as_view(), name='edit_org'),
     path('<int:org_id>/volunteer/', AddVolunteer, name='add_volunteer'),
+    path('<int:org_id>/group/', GroupView.as_view()),
     path('<int:org_id>/group/', GetGroup, name='get_group'),
```

Comment:
Could you explain why did you add another path with `'int:org_id>/group/'` . I feel confused 😅